test/unit/test_sign.py

```diff
@@ -21,20 +21,18 @@
 from sigstore._sign import Signer
 
 
+@pytest.mark.online
 def test_signer_production():
     signer = Signer.production()
     assert signer is not None
 
 
-def test_signer_staging():
+def test_signer_staging(mock_staging_tuf):
     signer = Signer.staging()
     assert signer is not None
 
 
-@pytest.mark.online
-@pytest.mark.ambient_oidc
-@pytest.mark.parametrize("signer", [Signer.production(), Signer.staging()])
```

Comment:
Is there a reason for removing this parametrization?

Reply:
Yes, the parametrization setup happens even if --skip-online: and that setup currently does a network request... 

Now that I think about it, I guess the parameter could be the class, and we only instantiate inside the test. That might work?

Reply:
That SGTM!